phase1/gce/DEVEL.md

```diff
@@ -0,0 +1,146 @@
+# Deploying a Kubernetes Development Cluster
+
+This describes the steps to use kubernetes-anywhere to deploy a custom build of
+kubernetes in GCE on a Linux that has the apt package manager.
+
+- gcp-project-id = the Google Cloud Platform project id where you want your dev
+  cluster to run.
+- gcs-bucket-name = GCS bucket you will create to hold some artifacts needed
+  during the node installation process.
+
+## Setup Steps
+
+1. Create a GCS bucket to hold your builds and make it public so that during the
+   installation process, when kubeadm runs on the GCE instances, it will be able
+   to access the bucket and download the debian packages that will be there.
+
+    ```sh
+    gsutil mb gs://<gcs-bucket-name>
+    gsutil defacl ch -u AllUsers:R gs://<gcs-bucket-name>
+    ```
+
+2. Modify the GCS path in `kubernetes-anywhere/phase1/gce/configure-vm-kubeadm.sh`
+   to be your personal GCS bucket created in the previous step.
+
+    ```diff
+    -KUBEADM_VERSION=$(get_metadata "k8s-kubeadm-version")
+    +KUBEADM_VERSION="gs://<gcs-bucket-name>/build/debs"
+    ```
+
+3. Right after that line, add a line setting the repo prefix.
+
+    ```diff
+    +export KUBE_REPO_PREFIX="gcr.io/<gcp-project-id>"
+    ```
+
+4. Further down in the same file, tell kubeadm to install your develpment build
+   instead of the default behavior, which is to install the most recent version.
+
+    ```diff
+       "master")
+    -    kubeadm init --discovery "token://${TOKEN}@" --skip-preflight-checks --api-advertise-addresses "$(get_metadata "k8s-advertise-addresses")"
```

Comment:
Yeah, absolutely. No need to target those kinds of changes in this PR, but they're great ideas for making the process much smoother (and making `kubernetes-anywhere` more useful).